portal-impl/src/com/liferay/portlet/internal/PortletQNameImpl.java

```diff
@@ -37,21 +37,6 @@ public String getKey(String uri, String localPart) {
 		return StringBundler.concat(uri, _KEY_SEPARATOR, localPart);
 	}
 
-	@Override
-	public String getPublicRenderParameterIdentifier(
```

Comment:
@slnn Can you keep this method? If removing, it needs to be removed together with `setPublicRenderParameterIdentifier` which still has usage. Please double check whether this identifier still makes any sense.

Reply:
Hi @dantewang 

I have double-checked, and we can remove only one usage of `setPublicRenderParameterIdentifier()` and then remove the method as no one will use the identifier after removing the `getPublicRenderParameterIdentifier()`